lib/home/widgets/recipes_screen.dart

```diff
@@ -18,26 +18,79 @@ class RecipesScreen extends StatelessWidget {
                 return Card(
                     child: ListTile(
                         title: Text(state.recipes[index].recipeName.value),
-                        subtitle: Text(state.recipes[index].recipeVolume.value),
+                        subtitle: Text(state.recipes[index].recipeType),
                         trailing: state.recipes[index].saved
                             ? const Icon(Icons.done)
                             : const Icon(Icons.rotate_left_rounded),
                         onTap: () => Navigator.pushNamed(
                             context, PageRouter.recipe,
-                            arguments: index)));
+                            arguments: index),
+                        onLongPress: () => {
+                              showDialog<String>(
+                                  context: context,
+                                  builder: (BuildContext context) =>
+                                      DeleteRecipe(
+                                          recipe: state.recipes[index]))
+                            }));
```

Comment:
In the long run, I think we should keep consistent design choices across the app, so we should decide on the style of deleting List Items — in `Collection` module I implemented buttons for this, here it's a long press, who knows what's better 😄!

I think for the initial test release of the app, we can keep both and gather feedback on what is preferred, so it actually might be good to leave it like it is 😊.

Reply:
Yeah, I did consider this - adding a button for each recipe/ingredient deletion just seemed like it could get cluttered quite easily so I decided to go this way. Like you said, would be good A/B testing I suppose but my approach does require prior knowledge of holding to delete etc